tests/unit/test_data_cleaning_workflow.py

```
@@ -0,0 +1,272 @@
+    original_age = original_data['age'].copy()
+    original_weight = original_data['weight'].copy()
+    
+    # Generate table
+    selected_vars = ['age', 'weight', 'outcome']
+    var_meta = {
+        'age': {'label': 'Age (years)', 'type': 'Continuous'},
+        'weight': {'label': 'Weight (kg)', 'type': 'Continuous'},
+        'outcome': {'label': 'Outcome', 'type': 'Categorical'}
+    }
+    
+    html_output = generate_table(
```

Ruff (0.14.13)

Local variable `html_output` is assigned to but never used

Remove assignment to unused variable `html_output`

(F841)